Pcap++/header/PcapRemoteDevice.h

```diff
@@ -91,8 +91,13 @@ namespace pcpp
 		std::shared_ptr<PcapRemoteAuthentication> m_RemoteAuthentication;
 
 		// c'tor is private, as only PcapRemoteDeviceList should create instances of it, and it'll create only one for
-		// every remote interface
```

Comment:
Why remove this line? 🤔

Reply:
huh, good catch. did not mean that to be removed.

Reply:
Fixed d9bef35